fronteerio/canvas-lms
# added live events for attachments + pages fix

test plan:
- create attachments by uploading course files
- update display name of course files
- delete some course files
- delete a course wiki page
- ensure appropriate messages are in the kinesis queue
